cartographer/mapping/map_builder.cc

```diff
@@ -205,18 +221,22 @@ void MapBuilder::LoadMap(io::ProtoStreamReader* const reader) {
       break;
     }
     if (proto.has_node()) {
-      const transform::Rigid3d node_pose =
-          node_poses.at(NodeId{proto.node().node_id().trajectory_id(),
-                               proto.node().node_id().node_index()});
-      sparse_pose_graph_->AddNodeFromProto(map_trajectory_id, node_pose,
-                                           proto.node());
+      const int new_trajectory_id =
+          trajectory_remapping.at(proto.node().node_id().trajectory_id());
+      proto.mutable_node()->mutable_node_id()->set_trajectory_id(
+          new_trajectory_id);
```

Comment:
Might be more readable to inline the remapping here and leave the `node_pose` code as before?

Reply:
Done